e2e/vitest.config.ts

```diff
@@ -6,7 +6,7 @@ import { defineConfig } from 'vitest/config';
 export default defineConfig({
   test: {
     include: ['./**/**/*.test.ts'],
-    testTimeout: 90_000,
+    testTimeout: 120_000,
```

Comment:
did it need more time?

thinking out loud I think this will increase our time waiting for test results by some non trivial amount? 
sometimes we see tests failing 3 times so this will add time

Reply:
Fixed 👍